docker/server/Dockerfile.ubuntu

```diff
@@ -80,15 +81,8 @@ RUN arch=${TARGETARCH:-amd64} \
     && mkdir -p /var/lib/clickhouse /var/log/clickhouse-server /etc/clickhouse-server /etc/clickhouse-client \
     && chmod ugo+Xrw -R /var/lib/clickhouse /var/log/clickhouse-server /etc/clickhouse-server /etc/clickhouse-client
 
-# Remove as much of Ubuntu as possible.
-# ClickHouse does not need Ubuntu. It can run on top of Linux kernel without any OS distribution.
-# ClickHouse does not need Docker at all. ClickHouse is above all that.
-# It does not care about Ubuntu, Docker, or other cruft and you should neither.
-# The fact that this Docker image is based on Ubuntu is just a misconception.
-# Some vulnerability scanners are arguing about Ubuntu, which is not relevant to ClickHouse at all.
-# ClickHouse does not care when you report false vulnerabilities by running some Docker scanners.
-
-RUN apt-get remove --purge -y libksba8 && apt-get autoremove -y
+RUN apt-get autoremove --purge -yq libksba8 && \
```

Comment:
Maybe it makes sense to put it in the same `apt-get session` with a comment to reduce the image's size?